lib/js/_publish/_add-files-to-package-json.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
I'm really not sure this is the best of the two options we have - the other being to create an `.npmignore` file. It seems far more risky to accidentally exclude a file that is needed by specifying a whitelist (this option), than to include a file that didn't really need to be included (using NPM ignore).

With this option you also have to read the `package.json` file before writing it back to disk, but with the `.npmignore` option you just write a file once.